testing/apps/details/Makefile

```diff
@@ -1,18 +1,15 @@
 APP_NAME 	:= $(shell basename `pwd`)
 IMAGE_NAME 	:= amalgam8/a8-testing-bookinfo-$(APP_NAME)
-LDFLAGS 	:=
 
-ifndef GOOS
-    GOOS := $(shell go env GOHOSTOS)
-endif
+BUILDFLAGS  := -i
+LDFLAGS 	:= -s -w -linkmode external -extldflags -static
 
-ifeq ($(GOOS),linux)
```

Comment:
The current setup of the integration test _requires_ the binary to be statically linked.
If we condition that on GOHOSTOS, then running this on non-linux will result in runtime error when launching the containers. So, we might as well fail it earlier during compile/link time.